client/state/login/reducer.js

```diff
@@ -62,6 +62,16 @@ export const isRequesting = createReducer( false, {
 	[ SOCIAL_DISCONNECT_ACCOUNT_REQUEST_SUCCESS ]: () => false,
 } );
 
+export const closeWindowAfterLogin = createReducer( null, {
```

Comment:
Do we really need to use `createReducer`? This seems to be a pretty straightforward usage that can be done in a pure function.